lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -448,6 +448,7 @@ namespace Js
         bool inProfileMode;
         bool inDispatchProfileMode;
         bool arrayObjectHasUserDefinedSpecies;
+        bool arrayObjectHasUserDefinedIterator;
```

Comment:
Removed in latest rev.
The species flag was there because we wanted to avoid any form of property look-up for perf reasons. So probably no.